stackstate_checks_base/stackstate_checks/base/checks/base.py

```diff
@@ -212,7 +212,8 @@ def _raise_unexpected_type(self, argumentName, value, expected):
         raise ValueError("Got unexpected {} for argument {}, expected {}".format(type(value), argumentName, expected))
 
     def _check_struct_value(self, argumentName, value):
-        if value is None or isinstance(value, string_types) or isinstance(value, integer_types):
+        if value is None or isinstance(value, string_types) or isinstance(value, integer_types) or \
+                isinstance(value, float) or isinstance(value, bool):
```

Comment:
Great, you are fixing the problem where it sits! 

Do we still need `clean_unsupported_metadata` then?

Reply:
Yeah because this is integration only but in main core agent it still needs to be fixed.